charts/nextcloud/values.yaml

```diff
@@ -3,9 +3,7 @@
 ##
 image:
   repository: nextcloud
-  flavor: apache
-  # If unset, uses flavor + .Chart.AppVersion to create tag
-  # tag:
+  tag: 24.0.3-apache
```

Comment:
This should not be needed anymore. You can simply update the appVersion from values.yaml